apps/src/netsim/NetSimRouterNode.js

```diff
@@ -1103,11 +1103,20 @@ NetSimRouterNode.prototype.forwardMessageToRecipient_ = function (message, onCom
 
   // TODO: Handle bad state where more than one wire matches dest address?
 
+  // Normally the recipient simulates a message.
```

Comment:
Do you plan to unify the logic so that the sender always simulates, or are there reasons to have the recipient continue to receive when there is no router, or is it just not worth the trouble to unify?